packages/integration-tests/src/test-project-setup/access_testing_project.ts

```diff
@@ -0,0 +1,520 @@
+        assert.strictEqual(
+          err.message,
+          'Not authorized to perform sts:AssumeRoleWithWebIdentity'
+        );
+        return true;
+      }
+    );
+  };
+
+  private createAuthenticatedSimpleAuthCognitoUser = async (
+    clientConfig: ClientConfigVersionTemplateType<'1'>
+  ): Promise<SimpleAuthCognitoUser> => {
+    if (!clientConfig.custom) {
+      throw new Error('Client config is missing custom section');
+    }
+    const simpleAuthUserPoolId = clientConfig.custom
+      .simpleAuthUserPoolId as string;
+    const simpleAuthUserPoolClientId = clientConfig.custom
+      .simpleAuthUserPoolClientId as string;
+    const simpleAuthIdentityPoolId = clientConfig.custom
+      .simpleAuthIdentityPoolId as string;
+    const username = `amplify-backend-${shortUuid()}@amazon.com`;
+    const temporaryPassword = `Test1@Temp${shortUuid()}`;
+    const password = `Test1@${shortUuid()}`;
+    await this.cognitoIdentityProviderClient.send(
+      new AdminCreateUserCommand({
+        Username: username,
+        TemporaryPassword: temporaryPassword,
+        UserPoolId: simpleAuthUserPoolId,
+        MessageAction: 'SUPPRESS',
+      })
+    );
+
+    const initiateAuthResponse: InitiateAuthCommandOutput =
+      await this.cognitoIdentityProviderClient.send(
+        new InitiateAuthCommand({
+          ClientId: simpleAuthUserPoolClientId,
+          AuthFlow: 'USER_PASSWORD_AUTH',
+          AuthParameters: {
+            USERNAME: username,
+            PASSWORD: temporaryPassword,
+          },
+        })
+      );
+
+    const respondToAuthChallengeResponse: RespondToAuthChallengeCommandOutput =
+      await this.cognitoIdentityProviderClient.send(
+        new RespondToAuthChallengeCommand({
+          ClientId: simpleAuthUserPoolClientId,
+          ChallengeName: 'NEW_PASSWORD_REQUIRED',
+          Session: initiateAuthResponse.Session,
+          ChallengeResponses: {
+            USERNAME: username,
+            NEW_PASSWORD: password,
+          },
+        })
+      );
+
+    const logins: Record<string, string> = {};
+    if (respondToAuthChallengeResponse.AuthenticationResult?.IdToken) {
+      logins[
+        `cognito-idp.${await this.cognitoIdentityClient.config.region()}.amazonaws.com/${simpleAuthUserPoolId}`
+      ] = respondToAuthChallengeResponse.AuthenticationResult.IdToken;
+    }
+    const getIdResponse: GetIdCommandOutput =
+      await this.cognitoIdentityClient.send(
+        new GetIdCommand({
+          IdentityPoolId: simpleAuthIdentityPoolId,
+          Logins: logins,
+        })
+      );
+
+    const getOpenIdTokenResponse: GetOpenIdTokenCommandOutput =
+      await this.cognitoIdentityClient.send(
+        new GetOpenIdTokenCommand({
+          IdentityId: getIdResponse.IdentityId,
+          Logins: logins,
+        })
+      );
+
+    if (!getOpenIdTokenResponse.Token) {
+      throw new Error('Missing OpenId Token');
+    }
+
+    return {
+      username,
+      password,
```

Comment:
looks like the caller only uses the token. Could we just return that and not the username and pw?